src/App.css

```diff
@@ -1,22 +1,20 @@
 .App {
   text-align: center;
+  display: grid;
+  grid-template-rows: 90vh, 10vh;
```

Comment:
The future is here! Great way to set up the app for a more declarative and predictable layout as it grows!